sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/protocol/SdkVersion.kt

```diff
@@ -0,0 +1,20 @@
+package io.sentry.kotlin.multiplatform.protocol
+
+actual data class SdkVersion actual constructor(
+    actual val name: String,
+    actual val version: String,
+    actual val packages: MutableList<Package>
+) {
+    fun toCocoaSdkVersion(): Map<Any?, *> {
```

Comment:
```suggestion
    fun toCocoaSdkVersion(): Map<String, Any> {
```

We can be more specific here

Reply:
This works but requires casting like `toCocoaSdkVersion() as Map<Any?, *>` when setting the sdk version on the cocoa sdk.

Reply:
I believe the data class should have the correct type, when calling `toCocoaSdkVersion` you can do the casting.